src/NoActiveAccessTokenException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
```

Comment:
Thanks, It's added now. Everything should be up-to-snuff as soon as @lucadegasperi gives feedback :+1: